### PR TITLE
feat: add ageGroup Field in attendee form

### LINKED ISSUE #3796: Attendee Form: capture age group
The current Attendee Form doesn't capture the age group, please add an additional required field in the form

Question: Select your age group 
Answers: Dropdown (required)
- 19 or less
- 20 to 29 
- 30 to 39
- 40 to 49
- 50 or above

### PR DESCRIPTION
**Pull request by snitin315**
Fixes #3796 

Related Server PR [#6744](https://github.com/fossasia/open-event-server/pull/6744)

#### Short description of what this resolves:
Added `ageGroup` Field in `Attendee Form`

#### Changes proposed in this pull request:

Screenshot from 2020-01-14 21-24-25

Screenshot from 2020-01-14 21-27-06

### CODE REVIEW

File: app/templates/components/forms/orders/order-form.hbs [hunk 1]
```diff
@@ -64,18 +64,32 @@
...
-                  <div class="default text">{{t 'Select your gender'}}</div>
-                  <div class="menu">
-                    {{#each genders as |gender|}}
-                      <div class="item" data-value="{{map-value mapper gender.name}}">
-                        {{gender.name}}
-                      </div>
-                    {{/each}}
-                  </div>
-                {{/ui-dropdown}}
+              {{#if (eq field.type 'select')}}
```
Comment (kushthedude) [inline app/templates/components/forms/orders/order-form.hbs]: Remove this if

File: app/models/attendee.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ export default ModelBase.extend({
   gender             : attr('string'),
   birthDate          : attr('moment'),
   complexFieldValues : attr(),
+  ageGroup           : attr('string'),
```
Comment (kushthedude) [inline app/models/attendee.js]: Is the field added in server ?

**Comment by kushthedude** [comment]
@snitin315 Please mention the link of merged server PR

**Comment by snitin315** [comment]
> @snitin315 Please mention the link of merged server PR

Related Server PR [#6744](https://github.com/fossasia/open-event-server/pull/6744)

File: app/utils/dictionary/age-groups.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export const ageGroups = [
+  {
+    name: '19 or less'
+  },
+  {
+    name: '20 to 29'
```
Comment (kushthedude) [inline app/utils/dictionary/age-groups.js]: Replace name with age, will make more sense.

Comment (snitin315) [reply app/utils/dictionary/age-groups.js]: Done

Comment (kushthedude) [reply app/utils/dictionary/age-groups.js]: Great!

**Pull request merged**